clang/include/clang/Format/Format.h

```diff
@@ -2218,6 +2218,11 @@ struct FormatStyle {
   /// \version 3.8
   bool BreakAfterJavaFieldAnnotations;
 
+  /// If ``true``, clang-format will always break before function definition
+  /// parameters
```

Comment:
```suggestion
  /// parameters.
```

Please also provide an example with on and off.